libraries/AP_Param/AP_Param.cpp

```diff
@@ -38,7 +38,7 @@ extern const AP_HAL::HAL &hal;
 #define ENABLE_DEBUG 1
```

Comment:
I don't know but it was requested to be keep enable : see https://github.com/ArduPilot/ardupilot/pull/5531